docs/pascal_voc_user_manual.md

```diff
@@ -0,0 +1,442 @@
+[user manual](../docs/user_manual.md)
+to get more information about these operations.
+
+There are few examples of using datumaro operations to solve
+particular problems:
+
+- We can convert Pascal VOC dataset to Market-1501 format. But Market-1501 dataset
+only has a `person` class, marked with a bounding box. And to perform the conversion
+we could filter the Pascal VOC dataset. With datumaro we can do it like this
+
+``` bash
+# create datumaro project with Pascal VOC dataset
+datum import -o <path/to/voc/dataset> --name project
+
+# convert labeled shapes into bboxes
+datum transform -t shapes_into_boxes
+cd project-shapes_to_boxes
+
+# keep only person class items
+datum filter --mode items+annotations \
+    -e '/item[label="person"]' \
+    -o <path/to/output/project>
+
+# delete other labels from dataset
+datum trasnform -t remap_labels -- -l person:person --default delete \
+    -o <path/to/final/project>
+```
+
+To make sure that the converting was succesful we can check the output project:
+
+```bash
+cd <path/to/final/project>
+datum info
+```
+
+- Pascal VOC 2007 use about 900MB disk space, you can store half as much if keep
+only store the test subset, and with datumaro split the test subset into test
+and training subsets:
+
+```bash
+# create datumaro project with Pascal VOC 2007 (only test subset) dataset
+datum import -o ./VOC2007 --name project
+
+# split the test subset into test and training subsets
+datum transform -t random_split
+
+# or you can specify ratio for subsets (by default test:.67 train:.33)
+datum transform -t random_split -- -s train:.5 -s test:.5
+```
+
+- If you don`t need a variety of classes in Pascal VOC dataset,
+with datumaro you can group the classes for your task:
+
+```bash
+# create datumaro project with Pascal VOC dataset
+datum import -o ./VOC2007 --name project
+
+# group the classes
+datum transform -t remap_labels -- -l car:vehicle -l aeroplane:vehicle \
+    -l bicycle:vehicle -l boat:vehicle -l bus:vehicle -l car:vehicle \
+    -l train:vehicle -l motorbike:vehicle -l bottle:indoor -l chair:indoor \
+    -l diningtable:indoor -l pottedplant:indoor -l sofa:indoor \
+    -l tvmonitor:indoor -l bird:animal -l cat:animal -l dog:animal \
+    -l horse:animal -l sheep:animal -l cow:animal -l person:person \
+    --default delete
+```
+
+- When choosing a dataset for research, it is often useful to find out how the
+datasets differ from each other, to see information about this difference, you
+can run `datum diff`. For example calculate difference between Pascal VOC 2007
+and Pascal VOC 2012:
+
+```bash
+# create project with Pascal 2007 dataset
+datum import -p ./project2007 -f voc <path/to/voc/2007>
+
+# create project with Pascal 2012 dataset
+datum import -p ./project2012 -f voc <path/to/voc/2012>
+
+# calculate difference
+datum diff -p ./proect2012 ./project2007
+
+Datasets have different lengths: 14974 vs 34314
+Unmatched items in the first dataset: {('00973', 'train'), ...}
```

Comment:
It will be good to show that the final result actually matches the expected difference, which is shown in the official dataset description. Now it is just a result with no meaning.